### DISCUSSION TITLE
will missing CVD-BTs be patches into former Releases after being published in SDK-1.9

### DISCUSSION THREAD

**Discussion by gabrieledall** [OP-TED/eForms-SDK#530] [Q&A]
Hello Team, 

as it was mentioned in the last TED-technical meeting that some missing CVD-fields will be added to the NTD with SDK-1.9 there is one question left to us. Will these fields will later be added to the former releases that are supported in the notices-Plattform?

Thank you for your answer.

**Comment by rkottmann** [comment]
Patching 1.5 and 1.7 with CVD fixes would tremendously support German adoption!
